app/Http/Controllers/Auth/LoginController.php

```diff
@@ -61,6 +61,11 @@ public function redirectToProvider($provider)
     public function handleProviderCallback($provider)
     {
         $user = Socialite::driver($provider)->user();
+
+        if(!isset($user->user['url'])) {
+            $user->user['url'] = NULL;
```

Comment:
nếu ở cái $user kia ko có user['url'] thì nó chả về ntn. nếu nó trả về là null rồi thì ko cần đoạn này, còn ko có thì ms cần chứ. với có cần dùng isset ko. 
Nếu có vs không có thì chỉ cần if($user->user['url']) là đc chứ nhỉ. 
còn nếu có trả về nhưng null thì cần gì?

Reply:
Cái này là ở bên google nó k có user->user['url'] nên nó sẽ ko trả về null, mà nó như kiểu là not found ấy.

Reply:
Thế dùng  if($user->user['url']) có đc ko.